main.tf

```diff
@@ -1,7 +1,21 @@
+data "aws_ami" "ecs_ami" {
+  most_recent = true
+
+  filter {
+    name = "owner-alias"
+    values = ["amazon"]
+  }
+
+  filter {
+    name = "name"
+    values = ["amzn-ami-*.*.g-amazon-ecs-optimized"]
```

Comment:
In order to make it more flexible you can replace it with: `values = ["amzn-ami-${var.ami_version}.g-amazon-ecs-optimized"]` and define variable like this:
```
variable "ami_version" {
  default = "*"
}
```
and update example code in README.

This way users can provide `ami_version` to the module, if they don't want to have unexpected upgrades.